status-monitor.lic

```diff
@@ -130,6 +130,7 @@ class GameFilter
   end
 
   def command_check(line)
+    line = line.gsub(/_|~|-|=|\./,"")
```

Comment:
I'd prefer to see the `.gsub` in the method chain on the next line. What do you think?

Reply:
Makes sense.  Updated